changelogs/fragments/remove_deprecated_modules.yaml

```diff
@@ -0,0 +1,3 @@
+---
+major_changes:
+  - Remove deprecated Junos Modules.
```

Comment:
Can we list all the modules that were removed?

Reply:
Updated. Thanks